static/topbar.js

```diff
@@ -39,11 +39,12 @@ const mapStateToProps = (state) => {
             break;
         case BuildState.BUILT:
             visibleBrowseLink = true;
+            visibleHomeLink = false;
+            buildState = BuildState.BUILT;
             break;
         case BuildState.BUILT_AND_NAVIGATING:
             visibleBrowseLink = true;
             visibleHomeLink = true;
-            buildState = BuildState.BUILT;
```

Comment:
iirc, we rely on this elsewhere in the TopBar (i.e., I don't check for the `BUILT_AND_NAVIGATING` state, just the `BUILT` one (this should be documented, sorry).